File: PYTHON/chapter 3/user_input.py
number = int(input("Enter a number: "))
passes = 0
failures = 0

while number != 1 and number != 2:
	number = int(input("Enter a number: "))
	failures = failures + 1
number+=1

if number == 1 or number == 2:
	passes = passes + 1
print(f'passed: {passes}')
print(f'failed: {failures}')

File: JAVA/Chapter4/UserInput.java
import java.util.Scanner;
	public class UserInput{
	public static void main(String[]args){
	Scanner input = new Scanner(System.in);

	int count = 0;

	System.out.println("Enter a number: ");
	count = input.nextInt();

	while(count !=1 && count !=2){
	System.out.println("Enter a number: ");
	count = input.nextInt();

	}
	System.out.println("You won");
}
}